Description: rusr/lib/paraview/libvtkXdmf.so contains an unresolvable 
 reference to symbol XtDispatchEvent.
Author: Mathieu Malaterre <mathieu.malaterre@gmail.com>
Bug: http://paraview.org/Bug/view.php?id=12576
Last-Update: 2011-09-07

--- a/Utilities/Xdmf2/vtk/CMakeLists.txt
+++ b/Utilities/Xdmf2/vtk/CMakeLists.txt
@@ -105,6 +105,7 @@
   ELSE(VTK_USE_PARALLEL)
     TARGET_LINK_LIBRARIES(vtkXdmf vtkIO Xdmf vtkRendering)
   ENDIF(VTK_USE_PARALLEL)
+  TARGET_LINK_LIBRARIES(${X11_Xt_LIB})
 
   IF (VTK_USE_MPI)
     # Needed for mpich 2
